### PR TITLE
Cookie and privacy

### PR DESCRIPTION
**Pull request by JOE-SNWPLW**

### CODE REVIEW

File: src/pages/terms-and-conditions.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Terms and conditions"
+date: "2020-11-24"
+---
+
+# Terms and conditions
+
+Paying customers of Snowplow Analytics Ltd. and Snowplow Analytics Inc. may download, make copies of and distribute the technical documentation that we publish on this wiki internally within their business as reasonably necessary in connection with their agreed use of our paid-for products.
```
Comment (stanch) [inline src/pages/terms-and-conditions.md:8]: Let me review this with legal, I think we can simplify this.

**Review by stanch — CHANGES_REQUESTED** [review]

File: src/pages/terms-and-conditions.md (hunk 1 above)
Comment (stanch) [reply src/pages/terms-and-conditions.md:8]: Please replace this entire text with just:

---

The documentation on this site is published under the terms of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International Public Licence available at:

[https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.)

**Review by stanch — CHANGES_REQUESTED** [review]

File: src/pages/cookie-settings.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    <tr>
+      <td>_ga</td>
+      <td>The _ga cookie, installed by Google Analytics, calculates visitor, session and campaign data and also keeps track of site usage for the site's analytics report. The cookie stores information anonymously and assigns a randomly generated number to recognize unique visitors.</td>
+    </tr>
+    <tr>
+      <td>_gid</td>
+      <td>Installed by Google Analytics, _gid cookie stores information on how visitors use a website, while also creating an analytics report of the website's performance. Some of the data that are collected include the number of visitors, their source, and the pages they visit anonymously.</td>
+    </tr>
+    <tr>
+      <td>_gat_gtag_UA_159566509_1</td>
```
[the remark this answers is not in the document]
Comment (JOE-SNWPLW) [reply src/pages/cookie-settings.md]: I saw this after adding google tracking on localhost.

File: src/css/custom.css [hunk 3]
```diff
@@ -161,8 +161,9 @@ p {
   line-height: 24px;
 }
 
-.gh-button:active, .gh-button:hover, .gh-button:focus {
-  color: #412476;
+.snwpl-nav-button:active, .snwpl-nav-button:hover, .snwpl-nav-button:focus {
```
[the remark this answers is not in the document]
Comment (JOE-SNWPLW) [reply src/css/custom.css:164]: This class is added only to a specific instance of the NavItem through the docusaurus config file, so I didn't see a clean way to add it using the generic swizzling approach.

File: src/pages/cookie-settings.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <tr>
+      <td>_ga</td>
+      <td>The _ga cookie, installed by Google Analytics, calculates visitor, session and campaign data and also keeps track of site usage for the site's analytics report. The cookie stores information anonymously and assigns a randomly generated number to recognize unique visitors.</td>
+    </tr>
+    <tr>
+      <td>_gid</td>
+      <td>Installed by Google Analytics, _gid cookie stores information on how visitors use a website, while also creating an analytics report of the website's performance. Some of the data that are collected include the number of visitors, their source, and the pages they visit anonymously.</td>
+    </tr>
+    <tr>
+      <td>_sp5_</td>
```
[the remark this answers is not in the document]
Comment (JOE-SNWPLW) [reply src/pages/cookie-settings.md:45]: Yes, markdown can be mixed with html.

[the remark this answers is not in the document]
Comment (JOE-SNWPLW) [reply src/pages/cookie-settings.md:45]: The structure of the table with different col spans is not supported to be written in simple markdown and I didn't want to bother looking for extensions and extra configuration (if it's even available for mdx + docusaurus).

[the remark this answers is not in the document]
Comment (stanch) [reply src/pages/cookie-settings.md:56]: My concern is whether `_sp5_` is going to render in italics and without the underscores :)

File: src/pages/cookie-settings.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="2">Necessary</th>
+    </tr>
+    <tr>
+      <td>cookie-preferences</td>
+      <td>Ironically, to respect your cookie preferences we have to set a cookie for storing and processing them.</td>
```
Comment (stanch) [inline src/pages/cookie-settings.md:38]: Makes me wonder why these plugins don’t use localStorage...

Comment (JOE-SNWPLW) [reply src/pages/cookie-settings.md:38]: Maybe they want to be by default sharing the preferences with the server.

**Review by stanch — APPROVED** [review]

**Pull request merged**